### PR TITLE
[Feature] Add User Service

### PR DESCRIPTION
**Pull request by gywls517**
#31

### CODE REVIEW

File: src/components/organisms/GHeader.tsx [hunk 1]
```diff
@@ -1,29 +1,20 @@
...
-    removeCookie('authtoken');
-    removeCookie('refreshtoken');
-    router.push('/login');
-  };
-
   const dropDownMenu = (
     <Menu style={{width: 90, fontWeight: 300}}>
       <Menu.Item key="0">내 정보</Menu.Item>
-      <Menu.Item key="1" onClick={handleLogout}>
+      <Menu.Item key="1" onClick={() => UserService.logout()}>
```
Comment (orange-mush) [inline src/components/organisms/GHeader.tsx]: 😂

File: src/lib/services/User.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  Router.push('/login');
+};
+
+export const refresh = () => {
+  base()
+    .post('/partner/token/refresh/', {refresh: cookies.get('refreshtoken')})
+    .then(res => {
+      setCookie('authtoken', res.data.access);
+      setCookie('refreshtoken', res.data.refresh);
+      Router.push('/dashboard');
```
Comment (orange-mush) [inline src/lib/services/User.ts]: 😂

File: src/components/organisms/GHeader.tsx (hunk 1 above)
Comment (gywls517) [reply src/components/organisms/GHeader.tsx]: 050d656

File: src/lib/services/User.ts (hunk 2 above)
Comment (gywls517) [reply src/lib/services/User.ts]: 448def6

**Pull request merged**